torch/__init__.py

```diff
@@ -325,6 +325,12 @@ def manager_path():
 from . import _torch_docs, _tensor_docs, _storage_docs
 del _torch_docs, _tensor_docs, _storage_docs
 
+def check_env_flag(name, default=''):
+    return os.getenv(name, default).upper() in ['ON', '1', 'YES', 'TRUE', 'Y']
+
+# Prototypes overwrites torch builtins with functions!
+if check_env_flag('USE_PROTOTYPES'):
```

Comment:
So we can build with USE_PROTOTYPES but also control the feature after it's been built.

Reply:
Yes. I want to make sure this interferes as little as possible with the current built.